src/agent/rustjail/src/container.rs

```diff
@@ -605,6 +607,14 @@ fn do_init_child(cwfd: RawFd) -> Result<()> {
         env::set_var(v[0], v[1]);
     }
 
+    // set the "HOME" env getting from "/etc/passwd"
+    if env::var_os(HOME_ENV_KEY).is_none() {
+        match dirs::home_dir() {
```

Comment:
Since `None` is ignored, might be the following chuck of code could be more clear:

```rust
if let Some(home_dir) = dirs::home_dir() {
    env::set_var(HOME_ENV_KEY, home_dir)
}
```

Reply:
Hi @devimc 
Updated.